gatsby-theme-landing-page/src/components/benefits.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import * as styles from "./benefits.module.css";
+import MarkdownText from "./markdown-text";
+import Link from "./link";
+
+export default function Benefits({ heading, secondaryHeading, content }) {
+  return (
+    <section>
+      <div className={styles.container}>
+        <h2 className={styles.heading}>{heading}</h2>
+        {secondaryHeading && (
+          <h3 className={styles.subhead}>{secondaryHeading}</h3>
```

Comment:
After we're done, we should do a pass and standardize the classnames we use for h2, h3, and content fields, since this is such a common pattern

Reply:
I'm mulling whether or not we should avoid using child selectors for these, but totally agree